https://github.com/kubernetes/dashboard/pull/3233

commit 35b0e02a7fffdb3e8290413f85445cb8e697e26b
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Sat Aug 25 12:39:39 2018 +0200

    Don't claim copyright for future years
    
    When building the kubernetes-dashboard openSUSE package
    in future years, files differed because they contained
    something like
    this.latestCopyrightYear="2033"
    
    See https://reproducible-builds.org/ for why this matters.
    
    And https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated suggests that expiry should not be a concern anyway.
    
    https://github.com/kubernetes/kubernetes/pull/59172 fixed this for kubernetes.

diff --git a/build/conf.js b/build/conf.js
index 55de47375..143861949 100644
--- a/build/conf.js
+++ b/build/conf.js
@@ -64,6 +64,10 @@ const version = {
    * Version name of the head release of the project.
    */
   head: 'head',
+  /**
+   * Year of last change of the project
+   */
+  year: '2018',
 };
 
 /**
diff --git a/build/script.js b/build/script.js
index 2ff598925..6fdd8b3bf 100644
--- a/build/script.js
+++ b/build/script.js
@@ -242,7 +242,7 @@ function patchBuildInformation() {
                   patterns: [
                     {match: 'BUILD_GIT_COMMIT', replacement: commit},
                     {match: 'BUILD_DASHBOARD_VERSION', replacement: conf.deploy.version.release},
-                    {match: 'BUILD_YEAR', replacement: new Date().getFullYear()},
+                    {match: 'BUILD_YEAR', replacement: conf.deploy.version.year},
                   ],
                 }));
 }
